dev/TeachingTip/TeachingTip_v2.5.xaml

```diff
@@ -581,22 +585,27 @@
                                                         FontStretch="{TemplateBinding FontStretch}"/>
                                                 <Grid Grid.Row="2">
                                                     <Grid.ColumnDefinitions>
-                                                        <ColumnDefinition Width="*"/>
+                                                        <ColumnDefinition x:Name="ActionButtonColumn" Width="*"/>
+                                                        <ColumnDefinition x:Name="ButtonCenterSpacingColumn" Width="{StaticResource TeachingTipCenterSpacingWidth}"/>
                                                         <ColumnDefinition Width="*"/>
                                                     </Grid.ColumnDefinitions>
                                                     <Button x:Name="ActionButton"
+                                                            Margin="{StaticResource TeachingTipActionButtonMargin}"
```

Comment:
This is just the top margin, can be moved to the grid containing the buttons - TailEdgeBorder

Reply:
Thought that we should keep up the customization options but having a resource for the containing panel instead of the individual buttons makes more sense.